plugins/cosmos-crypto/cosmos_crypto/cosmos.py

```diff
@@ -32,15 +32,14 @@
 from pathlib import Path
 from typing import Any, BinaryIO, Collection, Dict, List, Optional, Tuple, cast
 
-from bech32 import bech32_decode, bech32_encode, convertbits
-from ecdsa import SECP256k1, SigningKey, VerifyingKey
-from ecdsa.util import sigencode_string_canonize
-
+import requests
 from aea.common import Address, JSONLike
 from aea.crypto.base import Crypto, FaucetApi, Helper, LedgerApi
 from aea.exceptions import AEAEnforceError
-from aea.helpers import http_requests as requests
```

Comment:
this is not available in the latest published `aea` version, and this matters now.

Reply:
Ok, then we need to set explicit timeout for all requests usage though (like done in the helper)

Reply:
see 231adc0